flask-server/avenv/lib/python3.12/site-packages/click/_termui_impl.py

```
@@ -0,0 +1,847 @@
+
+    # If we're piping to less and the user hasn't decided on colors, we enable
+    # them by default we find the -R flag in the command line arguments.
+    if color is None and cmd_name == "less":
+        less_flags = f"{os.environ.get('LESS', '')}{' '.join(cmd_params)}"
+        if not less_flags:
+            env["LESS"] = "-R"
+            color = True
+        elif "r" in less_flags or "R" in less_flags:
+            color = True
+
+    c = subprocess.Popen(
```

Ruff (0.13.2)

`subprocess` call with `shell=True` identified, security issue

(S602)